qa/L0_model_config/autofill_noplatform/python/model_transaction_policy_invalid_args/config.pbtxt

```diff
@@ -0,0 +1,26 @@
+name: "model_transaction_policy_invalid_args"
```

Comment:
Let's remove this name (same for other configs)

Reply:
For context, I think this is the direction we're moving since the name should just be auto-completed anyway. This reduces unnecessary lines of code for our tests and make them less brittle.

CC: @rmccorm4

Reply:
I remember we had a discussion on this in code reviews, here I was following the style of tests. Need to verify one thing and will update